scarlet/frame.py

```diff
@@ -161,32 +153,33 @@ def convert_pixel_to(self, target, pixel=None):
         return pixel_
 
     @staticmethod
-    def from_observations(
-        observations, target_psf=None, target_wcs=None, obs_id=None, coverage="union"
-    ):
-        """Generates a frame from a set of observations.
+    def from_observations(observations, target_psf=None, obs_id=None, coverage="union"):
+        """Generates a suitable model frame for a set of observations.
 
-        By default, this method will generate a frame from a set of observations by indentifying the highest resolution
+        This method generates a frame from a set of observations by indentifying the highest resolution
         and the smallest PSF and use them to construct a common frome for all observations in the set.
 
         Parameters
         ----------
         observations: array of `scarlet.Observation` objects
             array that contains Observations to match onto a common frame
-        psfs: `scarlet.PSF` or its arguments
-            Target PSF to which oll observations are to be deconvolved.If set to None we use the smallest PSF across
-            all observations and channels.
-        wcs: TBD
```

Comment:
This would be contrary to the purpose of this method. It now creates a WCS for the model frame based on all of observations. It cannot accept another WCS instead. It could do is to accept a different pixel scale. Create an issue if you want that functionality.

Reply:
Why not, this is how it works now. Not only regarding pixel size, but also rotation and alignment. I don't see why we should remove it to put it back later in a custom version. What `from_observations` does, is to set a common frame for the observations and makes the process automated in case the user does not give anything, but if the user already knows what parameters to use for the frame, we should let them.

Reply:
I understand. This can be done only if we make the method ignore anything about `coverage` because you cannot define the spatial footprint *and* ask for consistent overlap calculations.

Reply:
Not sure I follow. This just requires an anchor point, which is specified in the wcs.

Reply:
Yes, you are right. I mistook the shape of the image to be part of the WCS. For us, it's the shape of the frame. I will re-enable the option.